model-optimizer/extensions/front/standalone_const_eraser.py

```diff
@@ -22,6 +22,9 @@
 
 class StandaloneConstEraser(FrontReplacementSubgraph):
```

Comment:
Do we really need this transformation during the front phase? Does it cause some issue if the Const->Result sub-graph reaches partial inference phase? Please, investigate this and may be this transformation could be removed.

Reply:
I would postpone this change until plugin teams give green signal. Now it does not affect MO work, size, etc.